ecosystem/sep-0045.md

```diff
@@ -0,0 +1,431 @@
+the **Server** domain.
+
+#### Request
+
+```
+GET <WEB_AUTH_ENDPOINT_SEP0045>
+```
+
+##### Request Parameters:
+
+| Name            | Type          | Description                                                                                                                                                                                                                                                                                                                                                                                                                |
+| --------------- | ------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `address`       | `C...` string | The **Client Account** address (`C...`) that the **Client** wishes to authenticate with the **Server**.                                                                                                                                                                                                                                                                                                                    |
+| `home_domain`   | string        | A **Home Domain**. Servers that generate tokens for multiple **Home Domain**s can use this parameter to identify which home domain the **Client** hopes to authenticate with.                                                                                                                                                                                                                                              |
+| `client_domain` | string        | (optional) a **Client Domain**. Supplied by **Clients** that intend to verify their domain in addition to the **Client Account**. See [Verifying the Client Domain](#verifying-the-client-domain). **Servers** should ignore this parameter if the **Server** does not support **Client Domain** verification, or the **Server** does not support verification for the specific **Client Domain** included in the request. |
+
+Example:
+
+```
+GET https://auth.example.com/?address=CCIBUCGPOHWMMMFPFTDWBSVHQRT4DIBJ7AD6BZJYDITBK2LCVBYW7HUQ&home_domain=example.com
+```
+
+#### Response
+
+##### Success
+
+On success the endpoint must return `200 OK` HTTP status code and a JSON object with the following fields:
+
+- `authorization_entries`: a list of XDR-encoded `SorobanAuthorizationEntry`. It contains an entry for each of the
+  **Client Account**'s signers in addition to a signed entry from the **Home Domain Address** and optionally the
```

Comment:
> It contains an entry for each of the **Client Account**'s signers

I would expect only a single entry for the client account, and that single entry can have multiple signatures attached to it. An entry authorizes an action by an account, not by a single signer.

Is there another reason it needs multiple signers?

Reply:
> Is there another reason it needs multiple signers?

Nope, this was a mistake. Jake had also pointed this out.